Source/IntegrationTests/TestFiles/LitTests/LitTest/ast/reveal/revealInExpression.dfy

```diff
@@ -15,15 +14,15 @@ method RevealExpressionScope()
 {
   hide *;
   var a := (reveal P; assert P(0); true);
-  assert P(1); // error
+  assert P(1); // no error, expression leaks.
```

Comment:
I hope one day it will be clear enough why it leaks, or it will be the default that it does not leak unless the reveal itself is marked explicitly as leaking.

Reply:
I'm still hoping the remove the need for expression statements, but we'll see whether that's possible and desired.